Source/Particles/PhysicalParticleContainer.H

```diff
@@ -341,6 +341,7 @@ protected:
     // particle and the time of the boosted frame.
     bool boost_adjust_transverse_positions = false;
     bool do_backward_propagation = false;
+    bool m_rz_random_theta = false;
```

Comment:
Do we need this line here? It seems that `bool m_rz_random_theta = true;` in `Source/Particles/WarpXParticleContainer.H ` is already sufficient, no?

Reply:
Ah that's right, I will remove the one in WarpXParticleContainer, to be consistent with other parameters.

Reply:
I think the one in WarpXParticleContainer was the one that you set to `true`. Since you now removed that one and use only the one defined here, I think we should set this one to `true` as well:
```suggestion
    bool m_rz_random_theta = true;
```